apis/v1alpha1/shared_types.go

```diff
@@ -67,7 +67,6 @@ type AdminNetworkPolicyPort struct {
 type Port struct {
 	// Protocol is the network protocol (TCP, UDP, or SCTP) which traffic must
 	// match. If not specified, this field defaults to TCP.
-	// +kubebuilder:default=TCP
```

Comment:
It's possible I might need to use https://github.com/kubernetes/apiextensions-apiserver/blob/aabbdff63fe0a4e940e7cbe3e5188c3b6e4b63bf/pkg/apiserver/schema/validation.go#L49 (`x-kubernetes-int-or-string: true`) instead ? Would be good to check with an api expert.

Reply:
no I think we are good here.